### PR TITLE
Add embargo to the edit form

### LINKED ISSUE #237: Add embargo to manage release section of work edit form
As part of this work, also edit the section heading and introductory text as seen in the [mock-up](https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/420954309).

Screen Shot 2020-10-09 at 8 05 11 AM

This should be stored in the `embargo_date` field.

See #118 for access after embargo.

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes #237 

## How was this change tested?

Screen Shot 2020-11-03 at 2 32 51 PM

### CODE REVIEW

File: app/controllers/works_controller.rb [hunk 1]
```diff
@@ -46,7 +46,9 @@ def work_params
                                  'created(1i)', 'created(2i)', 'created(3i)',
                                  'created_range(1i)', 'created_range(2i)', 'created_range(3i)',
                                  'created_range(4i)', 'created_range(5i)', 'created_range(6i)',
-                                 :created_edtf, :abstract, :citation, :access, :license, :agree_to_terms,
+                                 :created_edtf, :abstract, :citation, :access, :license,
+                                 :release, 'embargo_date(1i)', 'embargo_date(2i)', 'embargo_date(3i)',
```
Comment (peetucket) [inline app/controllers/works_controller.rb:50]: Sorry for the ignorance, can you point me towards any docs describing this syntax for attributes: `(1i)` etc...

**Comment by peetucket** [comment]
Screenshot would be nice for Amy to review/approve

File: app/controllers/works_controller.rb (hunk 1 above)
Comment (jcoyne) [reply app/controllers/works_controller.rb:50]: https://stackoverflow.com/questions/51959940/rails-date-select-parameter-permit

**Comment by jcoyne** [comment]
@peetucket I don't think Amy does (or should do) PR reviews. I think she should critique and react to what is on https://sul-h2-qa.stanford.edu/ as that is the truest representation of the service we are providing.  That said, I'm happy to provide a screenshot for you.

File: app/controllers/works_controller.rb (hunk 1 above)
Comment (peetucket) [reply app/controllers/works_controller.rb:50]: I see, I knew about the permitted attributes stuff, but hadn't seen these date fields split into year/month/day like this.

**Comment by peetucket** [comment]
> @peetucket I don't think Amy does (or should do) PR reviews. I think she should critique and react to what is on [sul-h2-qa.stanford.edu](https://sul-h2-qa.stanford.edu/) as that is the truest representation of the service we are providing. That said, I'm happy to provide a screenshot for you.

The review on sul-h2-qa sounds good, as it allows for a PO to review something with UI implications.

**Review by peetucket — APPROVED** [review]

**Pull request merged**